lib/oli_web/controllers/pow/session_html/new.html.heex

```diff
@@ -42,7 +42,7 @@
           >
             OR
           </div>
-          <%= form_for @changeset, @action, [as: :user], fn f -> %>
+          <%= form_for @changeset, Routes.authoring_pow_session_path(OliWeb.Endpoint, :create, request_path: ~p"/workspaces/course_author"), [as: :user], fn f -> %>
```

Comment:
This works and is a good solution for this login, however I think the real culprit here is the routes defined in `lib/oli_web/pow/author_routes.ex` for `after_sign_in_path` and `after_registration_path`

Reply:
@eliknebel Do you think we should keep the code as it was and modify the routes in those two functions (after_sign_in_path and after_registration_path) ???